tests/io_performance/image/README.md

```diff
@@ -22,15 +22,15 @@ Before starting the container, you need to [get a token on GitHub.com](https://d
 6. **FIO_BS** - Determining the block size. Default=4k,64k,1m. Optional parameter. For example: 4k,1m
 7. **FIO_JOBS** - Determining the count jobs in test. Optional parameter. Default=1,8. For example: 1
 8. **FIO_DEPTH** - Determining the io depth in test. Optional parameter. Default=1,8,16. For example: 1,8,32
-9. **FIO_TIME** - Duration of each test in sec. Default=60. Optional parameter.
+9. **FIO_TIME** - Duration of each test in sec. Default=60.
```

Comment:
Is it required to set? If so, what about default (is it correct to have "default" and "required" at the same time)? Please elaborate.

Reply:
Yes, this parameter is required.
It has default values if we don't specify it at all, but it cannot be empty. For this reason, it must be passed in when running tests. For this reason, it must be passed when running tests via Eden and not necessarily when passed to the metadata when starting a container.